src/main/views/find-choice.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "template.njk" %}
+{% block pageTitle %}
+    {{ "Search Option" | safe }}
+  {% endblock %}
+  {% block beforeContent %}
+  {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: "/"
+  }) }}
+  {% endblock %}
+{% block content %}
+
+
```

Comment:
try not to have more than 1 empty line, as i dont think the white spaces contribute to the njk file do they?